src/main/java/org/moltimate/moltimatebackend/util/StructureUtils.java

```diff
@@ -195,24 +197,18 @@ private static double rmsd(double[] point1, double[] point2) {
     }
 
     public static String residueName(Group group) {
-        return group.getChemComp().getThree_letter_code() + " " + group.getResidueNumber().toString();
+        return group.getPDBName() + " " + group.getResidueNumber()
+            .toString();
     }
 
     public static String ecNumber(Structure structure) {
-        try {
-            return structure.getEntityInfos()
-                    .stream()
-                    .filter(Objects::nonNull)
-                    .findFirst()
-                    .get()
-                    .getEcNums()
-                    .stream()
-                    .filter(Objects::nonNull)
-                    .findFirst()
-                    .get();
-        } catch (Exception e) {
-            return "unknown";
+        RCSBDescription description = RCSBDescriptionFactory.get(structure.getPDBCode());
+        for(RCSBPolymer polymer: description.getPolymers()){
+            if(polymer.getEnzClass() != null){
+                return polymer.getEnzClass();
+            }
         }
+        return "unknown";
```

Comment:
Can you make this a public final constant value so we can check if `ecNumber(...) == Constants.UNKNOWN_EC_NUMBER`